components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -99,15 +99,17 @@ export default class GroupsList extends React.PureComponent<Props, State> {
     public async previousPage(e: any): Promise<void> {
         e.preventDefault();
         const page = this.state.page < 1 ? 0 : this.state.page - 1;
-        this.setState({checked: {}, page, loading: true});
-        this.searchGroups(page);
+        this.setState({checked: {}, page, loading: true}, () => {
```

Comment:
Are the checked groups not persisted across pages navigations?

Reply:
This is a limitation of the current implementation where only the groups that are visible and in state can be managed (checked, linked/unlinked). I'll look into how feasible it is to persist group selection across page navigations and add it to this PR if possible.

Reply:
Yeah lets see if it's possible, if it's a lot of work we can create a separate ticket for it

Reply:
Created a ticket to look into this https://mattermost.atlassian.net/browse/MM-42290. Implementing this would also require input from design and it's a low use case so I've added it to our backlog for a future discussion.

Reply:
Thanks!